pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -585,6 +585,10 @@ type TiKVSpec struct {
 	// EnableNamedStatusPort enables status port(20180) in the Pod spec.
 	// If you set it to `true` for an existing cluster, the TiKV cluster will be rolling updated.
 	EnableNamedStatusPort bool `json:"enableNamedStatusPort,omitempty"`
+
+	// ScalePolicy is the scale configuration for TiKV
+	// +optional
+	ScalePolicy *ScalePolicySpec `json:"scalePolicy,omitempty"`
```

Comment:
`*ScalePolicySpec` => `ScalePolicy`

1. No need to use pointer for this struct, because the behavior of `null` and `default` is same.
2. There is no `ScalePolicyStatus`, so `Spec` suffix is not needed